README.md

```diff
@@ -1,2 +1,186 @@
+ - **name** (mandatory), the reducer name (for instance: `todos`)
+   - it's used to generate actions types
+   - it's used to retrieve informations from selectors
+
+Example:
+ - this reducer will use `id` as key field
+ - it's combined into `state.api.raw`
+ - its name is `todos`
+```es6
+import factory from 'trampss-redux-data-store'
+// factory(fieldKey)(path)(name)
+export default factory('id')('api.raw')('todos')
+```
+
+Data will be stored into `state.api.raw.todos`
+
+### reducer
+The previous factory returns a function which is a reducer.
+You just have to combine it like any other reducer :
+```es6
+import { createStore, combineReducers, compose, applyMiddleware } from 'redux'
+
+// import your reducer
+// (created by tramps-redux-data-store factory)
+import todos from './myTodosReducer'
+
+// create your Redux store as usual
+const store = createStore(
+  combineReducers({
+    // [other reducer]
+    api: combineReducers({
+      // [other reducer]
+      raw: combineReducers({
+        // import your reducer into api.raw
+        // since we configured this path
+        todos,
+      }),
+      // [other reducer]
+    }),
+    // [other reducer]
+  }),
+  /* your Redux middlewares */
+)
+
+export default store
+```
+
+### actions
+The factory returns a function (this is the reducer) that also contains actions and selectors as fields.
+Some generic actions are available. By now, it's not possible to add custom ones.
+
+Actions are:
+ - `set` : set an array of instances of your resource
+   - `set(<array>)`
+   - will generate this kind of Redux action:
+     - `{ type: 'SET_todos', payload: <array> }`
+ - `add` : add an instance of your resource
+   - `add(<instance>)`
+   - will generate this kind of Redux action:
+     - `{ type: 'GET_todos', payload: <instance> }`
+ - `del` : delete one instance of your resource by its key
+   - `del(<key>)`
+   - will generate this kind of Redux action:
+     - `{ type: 'DEL_todos', payload: <key> }`
+ - `reset` : reset the reducer (wipe all data)
+   - `reset()`
+   - will generate this kind of Redux action:
+     - `{ type: 'RESET_todos' }`
+
+Example, we set todos to our reducer:
+```es6
+// import your reducer
+// (created by tramps-redux-data-store factory)
+import todos from './myTodosReducer'
+
+// dispatch can be given by one of your middleware (redux-thunk, redux-saga, etc)
+// or it can be given by react-redux for example (mapDispatchToProps)
+dispatch(
+  // set todos
+  todos.set([
+    {
+      id: '1', // we set 'id' as key in the factory
+      visible: true,
+      label: 'My first todo',
+    },
+    {
+      id: '2',
+      visible: false,
+      label: 'This todo is done',
+    },
+  ])
+)
+
+```
+
+### selectors
+The factory returns a function (this is the reducer) that also contains actions and selectors as fields.
+Some generic selectors are available. By now, it's not possible to add custom ones.
+
+Selectors are:
```

Comment:
**Format :**
I think that it will be clearer to present API methods in a table (with signature and description)